internal/pkg/service/stream/storage/node/coordinator/filerotation/operator_test.go

```diff
@@ -27,11 +30,22 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
-func TestFileRotation(t *testing.T) {
-	t.Parallel()
+const fileExpirationDiff = time.Minute
 
-	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
-	defer cancel()
+type testState struct {
+	interval      time.Duration
+	importTrigger targetConfig.ImportTrigger
+	clk           *clock.Mock
```

Comment:
Personally, I would reverse the order, first I would put the tests in the code, and then these helpers.

Reply:
Can be reversed.